common/script/libs/achievements.js

```diff
@@ -0,0 +1,507 @@
+import i18n from '../i18n';
+
+let achievs = {};
+let index;
+
```

Comment:
So all of the logic for figuring out if a given user earned certain achievements would be OUTSIDE of the content api, correct?

The only info inside the content api would be the localization string keys, sprite name string, and maybe the achievement types?